std/build.sbt

```diff
@@ -1 +1,5 @@
 name := "std"
+
+libraryDependencies in ThisBuild ++=
```

Comment:
Doesn't the `in ThisBuild` here add this dependency to the entire build? If I run `sbt "project core" console` I can import `algebra.std._`.

Is there a reason to split the build config up like this (or even to have both a root-level `build.sbt` and `project/build.scala`)?